### PR TITLE
feat: create server with express

### PR DESCRIPTION
**Pull request by alishanti98**
relates #21

### CODE REVIEW

**Review by shathakh — APPROVED** [review]

File: server/routes/index.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import express, {Request, Response}from 'express'
+const Router = express.Router()
```
Comment (LinaYahya) [inline server/routes/index.ts:2]:
```suggestion
const router = express.Router()
```

File: server/app.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import dotenv from 'dotenv';
+import express from 'express';
+import cookieParser from 'cookie-parser';
+import compression from 'compression';
+import router from './routes'
+
+const app = express();
+app.set('port', process.env.PORT || 3500);
+dotenv.config();
```
Comment (LinaYahya) [inline server/app.ts:9]: remove to top level,
before calling express

File: server/app.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+app.use(express.json());
```
Comment (LinaYahya) [inline server/app.ts:11]: `app.use()` takes a series of middlewares as an array, 
so I prefer to use it in that way
```js
app.use([cookieParser(), compression(), ..etc]);
```

File: server/routes/index.ts (hunk 1 above)
Comment (LinaYahya) [inline server/routes/index.ts:2]:
```suggestion
const Router = express.Router()
```

**Review by LinaYahya — CHANGES_REQUESTED** [review]
Good Job Ali,
I would prefer to use morgan as a logger middleware try to search about and find away to use it
[morgan](https://github.com/expressjs/morgan#readme)

File: server/index.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import app from './app';
+
+const port:number = app.get('port');
+
+app.listen(port, () => {
```
Comment (LinaYahya) [inline server/index.ts:5]: let us sync sequelize before listening to the server

**Review by mohammedagha27 — APPROVED** [review]

**Review by naghamabuwarda — APPROVED** [review]

**Review by mohammedagha27 — APPROVED** [review]

**Review by LinaYahya — APPROVED** [review]

**Pull request merged**